src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -1418,6 +1424,7 @@ namespace PoweredInductionUnits {
 			}
 		}
 		if ( GetCurrentScheduleValue( PIU( PIUNum ).SchedPtr ) <= 0.0 ) UnitOn = false;
+		if ( ( GetCurrentScheduleValue( PIU( PIUNum ).FanAvailSchedPtr ) <= 0.0 || DataHVACGlobals::TurnFansOff ) && ! DataHVACGlobals::TurnFansOn ) UnitOn = false;
```

Comment:
@rraustad Not sure this is correct.  The PIU terminal unit could still pass through primary airflow even if the PIU fan is scheduled off.

Reply:
I'm not positive either but this line of code is in the CalcSeriesPIU so I figured if the fan was off there would always be 0 flow. The TurnFansOn/Off flags are passed to the fan, if these say to turn off the fan, then fan outlet flow rate = 0.

    if ( PIU( PIUNum ).Fan_Num == DataHVACGlobals::FanType_SystemModelObject ) {
        HVACFan::fanObjs[ PIU( PIUNum ).Fan_Index ]->simulate( _, PIUTurnFansOn , PIUTurnFansOff ,_ );
    } else if ( PIU( PIUNum ).Fan_Num == DataHVACGlobals::FanType_SimpleConstVolume ) {
        Fans::SimulateFanComponents( PIU( PIUNum ).FanName, FirstHVACIteration, PIU( PIUNum ).Fan_Index, _, PIUTurnFansOn, PIUTurnFansOff ); // fire the fan
    }
 
To be sure I traced the node connections through the SeriesPIU and see that primary and secondary air flow both pass through the fan. Regardless of real life PIU's, the E+ model does not allow flow through a fan when it is off.

![seriespiuschematic](https://user-images.githubusercontent.com/5778156/30571828-3656ea2c-9cb8-11e7-963f-6e6c7a38bf8c.png)